packages/account/src/Components/api-token/_tests_/api-token-delete-button.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { screen, render, fireEvent, waitFor, act } from '@testing-library/react';
+import ApiTokenContext from '../api-token-context';
+import ApiTokenDeleteButton from '../api-token-delete-button';
+
+const modal_root_el = document.createElement('div');
+modal_root_el.setAttribute('id', 'modal_root');
+document.body.appendChild(modal_root_el);
+
```

Comment:
Clarification: shoudn't this be in beforeAll hook?

Reply:
I think this can not be in beforeAll hook because it doesn't need to convert back to it's default value. We just need the modal to be created and append it to the root.